app/database/models/campaignModel.js

```diff
@@ -29,6 +29,53 @@ async function findById(id) {
   return campaign;
 }
 
+async function findAll(filters) {
+  const { skill } = filters;
+  try {
+    let campaigns = db('campaigns')
+      .join('users', 'users.id', 'campaigns.user_id')
+      .leftJoin('conservationists as cons', 'cons.user_id', 'users.id')
+      .select(
+        'users.profile_image',
+        'users.location',
+        'campaigns.*',
+        'campaigns.id as campaign_id',
+        'campaigns.name as camp_name',
+        'cons.name as org_name',
+      )
+      .where({ 'users.is_deactivated': false });
+
+    if (skill) {
+      campaigns = campaigns
+        .join(
+          'skilled_impact_requests',
+          'skilled_impact_requests.campaign_id',
+          'campaigns.id',
+        )
+        .select(
+          'skilled_impact_requests.skill',
+          'skilled_impact_requests.id as skilled_imact_request_id',
```

Comment:
Oops there's a typo here, it should be `impact`, not `imact`, for the `as skilled_imact_request_id`. My bad if I put that there originally :sweat_smile: